style/scss/_team.scss

```diff
@@ -18,12 +18,13 @@ $tablet-font-size-h4: 2rem;
 $tablet-line-height-p: 2rem;
 
 .team {
+  padding: 3rem 1rem;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
+  gap: 3rem;
   background-color: #f3f3f3;
-  padding: 0 3.2rem 2.4rem;
```

Comment:
The padding top should be consistent across all sections (7rem)

Reply:
Added 7 rem padding top to keep consistency over all sections